tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -365,6 +365,20 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         }
       }
     </style>
+    <template id="colabNotSupported" preserve-content>
+      <div style="max-width: 540px; margin: 80px auto 0 auto;">
+        <h3>Dynamic plugin isn’t supported in Colab yet.</h3>
```

Comment:
caps: `Iframe`, per the [standard camel case definition][1]

[1]: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
I have a feeling that this is a glitch with GitHub or something. Did you mean to leave this comment on this line?

Reply:
Hmm; nope. Odd.